Updated navigation and edits (#64)
PegaSysEng/ethsigner: docs/index.md

description: EthSigner is an open-source private transaction manager developed under the Apache 2.0 license and written in Java. 
<!--- END of page meta data -->

# EthSigner Transaction Signer

EthSigner is an open-source, client-agnostic, Ethereum transaction signer developed under the Apache 2.0 license and written in Java. 

> EthSigner enables key management to be seperated from transaction validation by providing private key signing. 

Reviewer: ```suggestion
EthSigner provides private key signing to separate key management from transaction validation. 
```
May I propose a simplified rephrase with more active form ?

Author: Much better - updated

## What can you do with EthSigner? 

> With EthSigner you can: 

Reviewer: ```suggestion 
EthSigner can: 
```

Author: Fixed

> * Sign Ethereum transactions 

Reviewer: ```suggestion
* Sign Ethereum transactions instead of your exposed node 
```

Author: I'm going to leave this for now until I follow up on the conversation with you, me, and Rob

> * Access your key store. 

Reviewer: ```suggestion
* Securely access your key store. 
```

Author: Same comment as above - need to dig in and find out whether there's additional requirements to make it secure
